explorer/app/components/Index/common/entities.ts

```diff
@@ -2,9 +2,15 @@
  * Set of possible metadata keys.
  */
 export enum METADATA_KEY {
+  ANATOMICAL_ENTITY = "ANATOMICAL_ENTITY",
   BIOSAMPLE_TYPE = "BIOSAMPLE_TYPE",
+  CONSENT_CODE = "CONSENT_CODE",
   DATA_MODALITY = "DATA_MODALITY",
+  DATA_TYPE = "DATA_TYPE",
   DATASET_NAME = "DATASET_NAME",
+  DEVELOPMENT_STAGE = "DEVELOPMENT_STAGE",
+  DISEASE_DONOR = "DISEASE_DONOR",
+  LIB_CONST_APPROACH = "LIB_CONST_APPROACH",
```

Comment:
```suggestion
  LIBRARY_CONSTRUCTION_APPROACH = "LIBRARY_CONSTRUCTION_APPROACH",
```

Reply:
I think we should use the full name for clarity - let's update any use of `LIB_CONST_APPROACH` here and elsewhere to `LIBRARY_CONSTRUCTION_APPROACH`.